files/ansible/deploy_mtt.yml

```diff
@@ -56,16 +63,23 @@
             - "{{ test_plans }}"
     when:
       - inventory_hostname in groups['sms']
-        
-  - name: Create directory because MTT
+
+  - name: Create Directory for Snapshot module template
     file:
-            path: "/opt/mtt/samples/python/"
+            path: "/opt/ohpc/pub/moduledeps/gnu8/openmpi4"
```

Comment:
hum, this is dependent on default compiler and mpi stack, can we pick that from somewhere else?

Reply:
AFAIK no we can't, Lmod is very picky it seems about where to look for its module files.